ci-operator/step-registry/baremetal/lab/pre/load-balancer/run/baremetal-lab-pre-load-balancer-run-commands.sh

```diff
@@ -79,6 +84,10 @@ for dev in "${devices[@]}"; do
     -pf "/var/run/dhclient.$interface.pid" \
     -lf "/var/lib/dhcp/dhclient.$interface.lease" "$interface"
 done
+
+nsenter -m -u -n -i -p -t "$(podman inspect -f '{{ .State.Pid }}' "haproxy-$CLUSTER_NAME")" \
+/sbin/dhclient -v -N
```

Comment:
What's the purpose of this command?

Reply:
it will trigger DHCPV6 negotiation in haproxy container to obtain IPv6 address

Reply:
see this:
https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/49805/rehearse-49805-periodic-ci-openshift-openshift-tests-private-release-4.15-arm64-nightly-baremetal-upi-ovn-dual-stack-f14/1774036430113214464/artifacts/baremetal-upi-ovn-dual-stack-f14/baremetal-lab-pre-load-balancer-run/build-log.txt

Reply:
This should be done earlier and together with the other two execution of dhclient.